### ISSUE TITLE
[bug]: When add or update something the list in the left is not update automatically added but need a refresh of the page

### ISSUE THREAD

**Issue by giosueDelgado** [hoppscotch/hoppscotch#3587]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

When I add a request, collection, update the name of some other action, the list is not updated automatically but needs a refresh of the page to update the section:  

![image](https://github.com/hoppscotch/hoppscotch/assets/11545723/366e512d-ee17-458e-a200-f6b9f750753b)

### Steps to reproduce

1. Add a collection:  
![image](https://github.com/hoppscotch/hoppscotch/assets/11545723/9b4668cf-c755-438d-b979-97d2d8648599)   

2. After click on save:  
![image](https://github.com/hoppscotch/hoppscotch/assets/11545723/a3a006cc-8876-4365-9279-ee30b5b1ffc0)

3. The list is not updated. After the refresh, I see that the list is updated correctly 

### Environment

Production

### Version

Self-hosted

**Comment by Xor767** [comment]
Same here, very annoying issue.
Even on the official cloud version of hoppscotch.io, it starts to happen when you sync your workspace.

**Comment by giosueDelgado** [comment]
Agree is very annoying issue because it seem there is something not work and then you refresh the page and appear! If is possible to give some priority I think is very common to see this issue

**Comment by Xor767** [comment]
I made it work by modifying VITE_BACKEND_WS_URL=wss://localhost:3170/graphql to VITE_BACKEND_WS_URL=ws://localhost:3170/graphql in the hoppscotch.env file

**Comment by KatyaGolyagina** [comment]
https://github.com/hoppscotch/hoppscotch/issues/3331 same

**Comment by giosueDelgado** [comment]
> #3331 same

Yes... I not found when I search, in this way can be closed in favor of #3331 
Thanks

**Comment by energyaccountnr1** [comment]
Faced the same issue. Works now with 9.1 and wss
